backend/app/configs/data_saver/jsonl_saver.yaml

```
@@ -4,4 +4,4 @@ defaults:
 
 name: jsonl_saver
 source: $kafka
-jsonl_file_path: ${oc.env:ROOT_PATH}/data_layer/database/db/jsonl_saver.csv
\ No newline at end of file
+jsonl_file_path: ${oc.env:ROOT_PATH}/app/data_layer/database/db/jsonl_saver.csv
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)